sycl/source/detail/graph_impl.hpp

```diff
@@ -66,6 +66,9 @@ struct node_impl {
 
   template <typename T>
   node_impl(graph_ptr g, T cgf) : MScheduled(false), MGraph(g), MBody(cgf) {}
+    
+  // empty or dummy node:
```

Comment:
Would prefer to just call this an empty node, "dummy" has ableist connotations so would avoid introducing the terminology